File: spec/code_samples/Python_python3/lists@cleaned/get.py
import http.client

conn = http.client.HTTPConnection("example.com")

headers = {
    'content-type': "application/json",
    'authorization': "Bearer REPLACE_BEARER_TOKEN"
    }

conn.request("GET", "/api/v1/lists/cleaned?username=SOME_STRING_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Php_http2/lists@cleaned/get.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('http://example.com/api/v1/lists/cleaned');
$request->setRequestMethod('GET');
$request->setQuery(new http\QueryString(array(
  'username' => 'SOME_STRING_VALUE'
)));

$request->setHeaders(array(
  'authorization' => 'Bearer REPLACE_BEARER_TOKEN',
  'content-type' => 'application/json'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();